app/src/components/AudioTimeline/TimelineScrollbar.tsx

```
@@ -0,0 +1,68 @@
+          className="absolute top-1 bottom-1 rounded-full bg-foreground/10 transition-colors hover:bg-foreground/15"
+          style={{ width: `${thumbWidth}px`, left: `${thumbLeft}px` }}
+        >
+          <div
+            role="slider"
+            aria-label="Zoom from left edge"
+            aria-valuenow={Math.round(pixelsPerSecond)}
+            aria-valuemin={Math.round(minPixelsPerSecond)}
+            aria-valuemax={Math.round(maxPixelsPerSecond)}
+            className="absolute top-0 bottom-0 left-0 w-1.5 cursor-ew-resize rounded-l-full bg-foreground/25 transition-colors hover:bg-foreground/40"
+            onMouseDown={onMouseDown('left')}
+          />
```

Biome (2.4.15)

[error] The HTML element with the interactive role "slider" is not focusable.

(lint/a11y/useFocusableInteractive)